package.json

```diff
@@ -107,7 +120,6 @@
   "devDependencies": {
     "@emotion/babel-plugin": "^11.9.2",
     "@emotion/eslint-plugin": "^11.7.0",
-    "@netlify/plugin-nextjs": "^4.30.0",
```

Comment:
Any reason that this has been moved from devDependencies to the ones contained in the production release?

Reply:
After I merged develop in this branch https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1678/commits/3186be2b539cb903c2365fd4b54fb5134a5555eb deployments started failing

Vecel was able to deploy once I moved netlify plugin away from devDependencies https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1678/commits/6b0475f026454adb5b00549cb446a82490b35efc